rest/config.go

```diff
@@ -485,6 +476,10 @@ func (dbConfig *DbConfig) validateVersion(isEnterpriseEdition bool) []error {
 		}
 	}
 
+	if dbConfig.DeprecatedPool != nil {
+		base.Warnf(`"pool" config option is deprecated and is not supported. This will be set the the default of "default" and the option should be removed from config gile.`)
```

Comment:
Typo in `gile`. Also nitpicking, but if we're not respecting it, is it fair to call it deprecated? Generally that would imply more of a "you can keep doing this for now, but it's going away...". Given that there's not a viable use case for it now, seems reasonably to say it's just not supported...

Reply:
@JFlath Yeah, thats fair enough. Will just say not supported. The reason the option itself is staying there is just to ensure that people that already have the option in their config doesn't have SGW fail to startup with "unknown field"

Reply:
Aye, absolutely. Suspected that was the reason we weren't just getting rid outright :)